air/src/proof.rs

```diff
@@ -120,7 +120,8 @@ impl ProofOptions {
     pub fn with_96_bit_security() -> Self {
         let options = WinterProofOptions::new(27, 8, 16, FieldExtension::Quadratic, 8, 255);
         Self {
-            hash_fn: HashFunction::Blake3_192,
+            // TODO: change back to blake
+            hash_fn: HashFunction::Rpo256,
             options,
         }
```

Comment:
If you rebase from #856, this change may not be needed as we should be able to pick RPO as the hash function by enabling the `recursive` flag.

For example, assuming the executable has been compiled with `make exec`, we can run the Fibonacci example with `-r` flag enabled to generate recursive proofs:
```
./target/optimized/miden example -r fib -n 200000
```